### PR TITLE
chore(ts): Convert adminBuffer

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: src/sentry/static/sentry/app/views/admin/adminBuffer.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import InternalStatChart from 'app/components/internalStatChart';
+
+const AdminBuffer = () => {
+  const since = new Date().getTime() / 1000 - 3600 * 24 * 7;
```
Comment (matejminar) [inline src/sentry/static/sentry/app/views/admin/adminBuffer.tsx:6]: Do we need to floor/round this? It will probably be something like `1604561186.888`

**Review by matejminar — APPROVED** [review]

File: src/sentry/static/sentry/app/views/admin/adminBuffer.tsx (hunk 1 above)
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/admin/adminBuffer.tsx:6]: It probably doesn't matter. I just left it what it was.

File: src/sentry/static/sentry/app/views/admin/adminBuffer.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+  return (
+    <div>
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/admin/adminBuffer.tsx:9]: should we replace this `div` with `React.Fragment`?

File: src/sentry/static/sentry/app/views/admin/adminBuffer.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        </div>
+      </div>
+
+      <div className="box">
+        <div className="box-header">
+          <h4>Updates Processed</h4>
+        </div>
+        <InternalStatChart
+          since={since}
+          resolution="1hr"
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/admin/adminBuffer.tsx:33]: Was the resolution not being passed down from somewhere?

**Review by priscilawebdev — APPROVED** [review]

File: src/sentry/static/sentry/app/views/admin/adminBuffer.tsx (hunk 3 above)
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/admin/adminBuffer.tsx:33]: It is not

File: src/sentry/static/sentry/app/views/admin/adminBuffer.tsx (hunk 2 above)
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/admin/adminBuffer.tsx:9]: Maybe, not sure, I didn't look too closely :)

**Pull request merged**